js-exercises/iter-infinite/iterInfinite.js

```diff
@@ -0,0 +1,33 @@
+function* count(start, step) {
```

Comment:
Would it work if the value passed to `start` or `step` is like '10'  numeric string or should it throw an error?